Dockerfile

```diff
@@ -54,7 +54,6 @@ USER root
 # For testing
 # hadolint ignore=DL3041
 RUN microdnf update -y \
-&& microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y java-1.8.0-openjdk-headless \
```

Comment:
I wonder if we should have weak_dependencies switched on for all these installs to reduce what packages we install?

Reply:
Done.